README.md

```diff
@@ -191,9 +196,12 @@ systemctl restart libvirtd
 
 ```bash
 cd tdx/guest-tools
-./td_virsh_tool.sh
+TD_IMG=<path_to_td_qcow2_image> ./td_virsh_tool.sh
```

Comment:
Since the README is getting more and more complex, from now on, i tend to simplify it by suggesting the user to use the default values (only command, no env variables, no arguments). We can specify the option like you do in the next section and if user wants to modify the behavior, one will need to set the appropriate variables,

Reply:
Sounds good. I just pushed a new commit with the shell variable removed from this command (and the qemu version). I also fixed a few typos that I noticed in the README while I was at it because I recently installed a spell checker in my IDE. :)